README.md

```diff
@@ -18,40 +18,99 @@ Some of these are tricky!  Don't trust your first instinct.
+Console says: False
+
+
 **f)** `x = true`
 
+True - Because you are declaring x as true.
+Console says: True
+
 **g)** `var x; x == 'undefined'`
 
+False - In the first step a variable, x, has been defined.
+Console says: false (because of why I say, I'm not so sure)
+
 **h)** `'9'<'10'`
 
+True - javascript will look at these strings as numbers for the operator
+Console says: false. Because in retrospect, javascript looks at the first digit in the string, rather than the string as a whole. Essentially this turns into 9<1, which is false.
+
 **i)** `typeof x + 1 === "number"`
 
+If x has been defined earlier as a number, this will output True.
+Console says: False. After tinkering around with what typeof x + 1 is on its own (after declaring x = 1), it retrieved "number1". So the console is actually adding a 1 to the type of x, which of course is not ultimately === "number"
+
+
 **j)** `typeof x % 2 === "number"`
 
+Similarly, if x has been defined earlier as a number, this will output True.
+Console says: False. I assume for the same reasons that I was wrong on the last example. I think the next example should still be true, however, because it is evaluating the typeof everything in the ()'s rather than the first thing it encounters.
+
+
 **k)** `typeof (x % 2) === "number"`
 
+Again, the same as i and j.
+Console says: True (though not by being the same as i and j!)
+
 **l)** `x++ == ++x`
 
+True, because though x (assuming it is a number) will output a different number on the console if these are placed in individually, they both store the same value in javascript. It is more a matter that x++ reflects the value of x and then adds one afterwards, while ++x adds one and then reflects its new value.
+Console says: False. I believe false because the order of operations is different and so the are not precisely equivalent, though I would have expected true with just double ='s. Perhaps my next prediction should have been false as well then.
+
 **m)** `++x == x++`
 
+
+True, because though x (assuming it is a number) will output a different number on the console if these are placed in individually, they both store the same value in javascript. It is more a matter that x++ reflects the value of x and then adds one afterwards, while ++x adds one and then reflects its new value.
+Console says: True. This is just a pile of nopes to me. I'll have to look at this again.
+
 **n)** `"1"+x == 1+x`
 
+True - Because of double ='s string 1 is seen as number 1 and added against x. This supposes x has been defined as a number.
+Console says: False. "1" == 1, and x == x, but "1"+x != 1+x. Perhaps the gaze of javascript does not squint through expressions.
+
 **o)** `"0"+1 == 1`
 
+True - Similar to above. In this case 0 is treated as a number, and added to 1 remains 1 and thus == 1.
+Console says: True. Though for my own logic..?
+
 **p)** `(typeof (x+1))===(typeof x)`	
```

Comment:
1p) If x is a string here, `typeof(x+1)` will return `string`, since 1 will autoconvert to a string, which makes this true with either a string or a number.